src/pynwb/base.py

```diff
@@ -50,26 +50,25 @@ def add_container(self, **kwargs):
         '''
         Add an NWBContainer to this ProcessingModule
         '''
-        warn(PendingDeprecationWarning('add_container will be replaced by add'))
+        warn("add_container is deprecated. Use add instead.", DeprecationWarning)
```

Comment:
Ah, I regret using `PendingDeprecationWarning` here. This should have been a `DeprecationWarning` a long time ago. Because Python ignores `PendingDeprecationWarning` by default, this had little value...